source/libs/inkutils.lua

```diff
@@ -20,6 +20,8 @@ local elapsedTime = nil
 function ink.resetElapsedTime()
     if playdate then
         playdate.resetElapsedTime()
+    elseif picotron then
+        elapsedTime = t()
     else
         elapsedTime = os.clock() * 1000
     end
```

Comment:
had to add specific `ink.resetElapsedTime` implementation that hooks into picotron's APIs